tools/README.md

```diff
@@ -0,0 +1,110 @@
+# Istio Load Testing User Guide
+This guide provides step-by-step instructions for using the `setup_perf_cluster.sh` load testing script.
+The script deploys a GKE cluster, an Istio service mesh, a GCE VM and runs [Fortio](https://github.com/istio/fortio/)
+on the VM and within the mesh. Fortio is used to perform load testing.
+
+### Clone Istio
+```
+$ git clone https://github.com/istio/istio.git && cd istio
```

Comment:
(not actionable now:) I think we should add the scripts to the release tar that way you don't need to clone just to do that

Reply:
+1

Reply:
#2874